lib/app_server.py

```diff
@@ -116,9 +116,9 @@ def __init__(self, _ini=None, test_suite=None):
     def logfile(self):
         # remove suite name using basename
         test_name = os.path.basename(self.current_test.name)
-        # add :conf_name if any
+        # add `-conf_name` if any
         if self.current_test.conf_name is not None:
-            test_name += ':' + self.current_test.conf_name
+            test_name += '-' + self.current_test.conf_name
```

Comment:
BTW, can we use `.` (dot) here? It is usually not used in test names.

Reply:
@ylobankov what do you think about it?

Reply:
I vote for `.`.

Reply:
Ok, fixed it :rocket: